mods/seasonal/abilities.js

```diff
@@ -91,8 +91,8 @@ let BattleAbilities = {
 	stimulatedpride: {
 		id: "stimulatedpride",
 		name: "Stimulated Pride",
-		desc: "On switch-in, this Pokemon summons Rain Dance, and it lowers the Attack of adjacent foes not behind a Substitute by one stage.",
-		shortDesc: "On switch-in, summons Rain Dance and lowers the Attack of adjacent foes by 1.",
+		desc: "On switch-in, this Pokemon lowers the Attack of adjacent foes not behind a Substitute by one stage. If Rain Dance is active, this Pokemon's Speed is doubled.",
+		shortDesc: "Switch-in: lowers Atk of adjacent foes by 1. 2x Speed in rain.",
```

Comment:
I should proably edit this again to make the description longer because abilities have more leeway.